src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -418,11 +415,11 @@ public void shouldNotSendPatronNoticeWhenCheckInNoticeIsDefinedAndCheckInDoesNot
       checkInResponse.getJson().containsKey("loan"), is(false));
 
     Awaitility.await()
-      .timeout(1, TimeUnit.SECONDS)
+      .pollDelay(1, TimeUnit.SECONDS)
```

Comment:
Is the expected behaviour for this check, to wait for 1 second then check the assertion, and failed if the assertion fails?

I'm not sure this is how this library works, won't it still poll until the timeout (implicit in this case)? And still pass eagerly if the first assert succeeds?

Reply:
This [topic](https://groups.google.com/forum/#!topic/awaitility/D1BXp9kprVI) suggests that Awaitility is not really intended to check for the absence of a change

Reply:
Yes, the expected behavior is what you described. Probably for this case (when we need to check that no notices has been sent) Awaitility is not the option.
And the simpler solution can be to use something like _Thread.sleep()_ and then check the condition.

Reply:
I think it is ok to use `thread.sleep` until we can determine an alternative.

I think the delay for checking the absence of a response in this test _should_ be the same as the checks for presence in order tests. A shorter delay, could lead to false positives for the absence checks.

This encourages a shorter limit for the presence checks, which helps to set constraints on the timeliness of the actual system.

Reply:
Done. Set 5 seconds as the delay